src/styles/main.scss

```diff
@@ -1,7 +1,74 @@
 @import "utils";
 @import "fonts";
 @import "typography";
+@import "blocks/header.scss";
+@import "blocks/slogan.scss";
+@import "blocks/comparison.scss";
+@import "blocks/details.scss";
+@import "blocks/contact.scss";
+@import "blocks/footer.scss";
 
-body {
-  background: $c-gray;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+}
+
+.page {
+  scroll-behavior: smooth;
+  
+  &__body {
+    background: $c-gray;
+    overflow-x: hidden;
+  }
+}
+
+.container {
+  padding: $mobile-container;
+}
+
+@media (min-width: 744px) {
+  .container {
+    padding: $tablet-container;
+    margin: 0 auto;
+  }
+}
+
+@media (min-width: 1260px) {
+  .container {
+    padding: $desktop-container;
+    max-width: $max-width;
+    margin: 0 auto;
+  }
+}
+
+.main {
+  background-color: $secondary-color;
+  padding: 60px 0 120px;
+
+  @media (min-width: 744px) {
+    padding: 120px 0;
+  }
+
+  @media (min-width: 1260px) {
+    padding: 120px 0 182px;
+  }
+}
+
+.d-none {
+  display: none;
+}
+
+.button {
+  border: unset;
+  transition: all $transition-duration;
```

Comment:
remember, it's not advised to use `transition: all` due to optimisation matters,
better specify certain props you transition